samples/dashboards/demo/datacursor-sync/demo.html

```diff
@@ -4,3 +4,7 @@
 <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
 
 <div id="container"></div>
+<p class="highcharts-description">
```

Comment:
Everything that is in the demo can be achieved through the API. 
Thus IMO it should not be included in the `demo` section. It might be only confusing to the users.
We can move it an create some sort of custom sync.

Reply:
@bre1470 any thoughts?

Reply:
It might be useful for development of custom components, but as a regular demo it is not helping much.

Reply:
I agree 😉 let's move it then